tests/hazmat/backends/test_openssl.py

```diff
@@ -434,6 +438,60 @@ def test_unsupported_oaep_label_decrypt(self):
                 )
             )
 
+    @pytest.mark.parametrize(
```

Comment:
This test isn't openssl specific, it should be with the rest of the generic RSA stuff.

Reply:
Whether or not it is supported is openssl specific (we're directly checking via a lib attribute), but since we don't really support alternate backends any more we can move this test into test_rsa easily.